### PR TITLE
Feat/material UI

### PR DESCRIPTION
**Pull request by Roshanjossey**
<h2>PR checklist</h2>

**Docs**
- [ ] If you added new code; is it self documenting? (all the variable names clear, comments added where appropriate)
- [ ] If you touched existing code; did you refactor existing naming, added comments where necessary?

**Tests**
- [ ] If your ticket adds new functionality; have you added unit tests to verify behaviour?
- [ ] If your ticket alters user interaction with the UI; have you added integration tests?
- [ ] Do your tests check for every _convievable_ user behabviour? (all behaviours the user _can_ perform)

**Typing**
- [ ] If you added new code; have you added typing?
- [ ] If you touched existing code; have you improved its typing?

**Styling**
- [x] If new styles were created; Did you refactor/add more styles in css-in-js style?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "@emotion/core": "^11.0.0",
     "@emotion/react": "^11.1.4",
     "@emotion/styled": "^11.0.0",
+    "@material-ui/core": "^5.0.0-alpha.29",
```
Comment (BastinJafari) [inline package.json:60]: while we are at it, shouldn't we delete bootstrap, react-bootstrap and shards-react?

**Review by BastinJafari — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Roshanjossey) [reply package.json:60]: I expect this to be an ongoing process. We have places where we're using react-bootstrap and shards now. If we remove them, we have to put a lot of effort into refactoring all the places where we use it. I think it's better to do this as we go along.

Comment (BastinJafari) [reply package.json:60]: Alrighty

**Review by BastinJafari — APPROVED** [review]
looks good!

**Pull request merged**